res/values/options.xml

```diff
@@ -279,5 +282,16 @@
 
     <string name="opt_superSlider_title">Super Slider</string>
     <string name="opt_superSlider_summary">Perfect shape slider!</string>
-
+	
+    <string name="opt_changespeed_title">Speed Modify</string>
+    <string name="opt_forcear_title">Force AR</string>
+    <string name="opt_enablenc_whenspeedchange_title">Enable NC in speed modify</string>
```

Comment:
I think this should be changed to "Enable metronome in speed modify" to avoid confusion with NC mod.

Reply:
fixed

Reply:
in fact, this option only change the tone of song. If enable metronome, it will cause speed abnormal bug(like NC bug in old version).